src/Cardano/CoinSelection.hs

```diff
@@ -6,12 +6,9 @@
 -- Copyright: © 2018-2020 IOHK
 -- License: Apache-2.0
 --
--- Provides the API of Coin Selection algorithm and Fee Calculation
--- This module contains the implementation of adjusting coin selection for a
--- fee.  The sender pays for the fee and additional inputs are picked randomly.
--- For more information refer to:
--- https://iohk.io/blog/self-organisation-in-coin-selection/
```

Comment:
As far as I can tell, this module doesn't actually include the implementation, and only includes the common API, so removing this comment!